jobs/kpi-forecasting/kpi-forecasting/yaml/desktop_non_cumulative_arima.yaml

```diff
@@ -0,0 +1,19 @@
+---
+target: desktop
+query_name: "desktop_qdau.sql"
+dataset_project: "mozdata"
+forecast_variable: "QDAU"
+forecast_parameters:
+  changepoint_prior_scale: 0.05
+  changepoint_range: 0.8
+  seasonality_prior_scale: 0.01
+columns:
+- submission_date
+- QDAU
+# write_project: "moz-fx-data-shared-prod"
+# output_table: "telemetry_derived.kpi_automated_forecast_v1"
+# confidences_table: "telemetry_derived.kpi_automated_forecast_confidences_v1"
+write_project: "moz-fx-data-bq-data-science"
+output_table: "pmcmanis.arima_results"
```

Comment:
Should this go into the prod dataset? or is the data science one correct here?

Reply:
Right now this needs to stay in data science. Until the new methodology is accepted by DS as the replacement, there just needs to be a valid yaml file so folks _could_ generate data if they wanted to.

Thank you for the eagle-eyed view though! It's genuinely appreciated.